### PR TITLE
Update generic class

### PR DESCRIPTION
**Pull request by axce1**
PR for #162

### CODE REVIEW

**Comment by lk-geimfari** [comment]
@axce1 Thanks! Please, look at Travis CI logs. It's seems that the tests failed.

**Comment by lk-geimfari** [comment]
@axce1 Also, please, do not forget add yourself to list of [contributors](https://github.com/lk-geimfari/elizabeth/blob/master/CONTRIBUTORS.md).

**Comment by axce1** [comment]
@lk-geimfari 
please, review my code

File: elizabeth/core/providers.py [hunk 1]
```diff
@@ -2659,6 +2655,12 @@ def __init__(self, locale):
         self.transport = Transport()
         self.path = Path()
 
+    def __getattr__(self, attrname):
```
Comment (sobolevn) [inline elizabeth/core/providers.py]: A doc string would be nice here.

**Comment by lk-geimfari** [comment]
@axce1 Look at review, please.

**Comment by lk-geimfari** [comment]
@axce1 Look, please: https://github.com/lk-geimfari/mimesis/issues/201

**Pull request merged**